FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,51 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+
+			guard self != nil else { return }
+
+			switch result {
+			case let .success((data, response)):
+
+				completion(FeedItemMapper.map(data, response: response))
+
+			case .failure:
+				completion(.failure(Error.connectivity))
+			}
+		}
+	}
+}
+
+internal final class FeedItemMapper {
+	private struct Root: Decodable {
+		let items: [Item]
+
+		var feed: [FeedImage] {
+			return items.map {
+				$0.item
+			}
```

Comment:
Suggestion: it's possible to use `map` with key paths to achieve the same result:
```suggestion
			return items.map(\.item)
```